packages/backend/src/app.ts

```diff
@@ -13,7 +13,10 @@ dotenv.config();
 const app = express();
 app.use(
   cors({
-    origin: '*',
+    origin:
+      process.env.NODE_ENV === 'development'
+        ? 'http://localhost:3000'
```

Comment:
Production link is: https://marvel-collector.vercel.app/ - please make this as an **env variable**

Also, please consider adding this to the production cors options for extra measure if possible:
```javascript
// Cache preflights to avoid extra preflight requests
maxAge: 600, // in seconds, max value allowed in Chrome
preflightContinue: false,
```

Reply:
Changes have been implemented.